src/overlays/actors/ovl_En_Hy/z_en_hy.c

```diff
@@ -29,71 +44,1023 @@ const ActorInit En_Hy_InitVars = {
     (ActorFunc)EnHy_Update,
     (ActorFunc)EnHy_Draw,
 };
-*/
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Hy/func_80A6F5B0.s")
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Hy/func_80A6F6C0.s")
+static ColliderCylinderInit D_80A71EC0 = {
+    { COLTYPE_UNK10, 0x00, 0x00, 0x39, 0x20, COLSHAPE_CYLINDER },
+    { 0x00, { 0x00000000, 0x00, 0x00 }, { 0x00000000, 0x00, 0x00 }, 0x00, 0x00, 0x01 },
+    { 20, 46, 0, { 0, 0, 0 } },
+};
+static CollisionCheckInfoInit2 D_80A71EEC = { 0x00, 0x0000, 0x0000, 0x0000, 0xFF };
+
+static UNK_PTR D_80A71EF8[] = { 0x060005C8, 0x06000DC8, 0x060015C8, NULL };
+static UNK_PTR D_80A71F08[] = { 0x0600057C, 0x0600067C, 0x0600077C, NULL };
+static UNK_PTR D_80A71F18[] = { 0x060004C8, NULL };
+static UNK_PTR D_80A71F20[] = { 0x060005FC, 0x060009FC, 0x06000DFC, NULL };
+static UNK_PTR D_80A71F30[] = { 0x060005FC, 0x060006FC, 0x060007FC, NULL };
+static UNK_PTR D_80A71F40[] = { 0x060007C8, 0x06000FC8, 0x060017C8, NULL };
+
+typedef struct {
+    /* 0x00 */ s16 objectId;
+    /* 0x04 */ Gfx* dList;
+    /* 0x08 */ UNK_PTR* unk_8;
+} EnHyUnknownStruct1; // size = 0xC
+
+static EnHyUnknownStruct1 D_80A71F50[] = {
+    { OBJECT_AOB, 0x06003C88, D_80A71EF8 }, { OBJECT_BOB, 0x06003B78, D_80A71F40 },
+    { OBJECT_BOJ, 0x060026F0, D_80A71F30 }, { OBJECT_BOJ, 0x060052E0, NULL },
+    { OBJECT_BOJ, 0x06005528, NULL },       { OBJECT_BOJ, 0x06005738, NULL },
+    { OBJECT_BOJ, 0x060059B0, NULL },       { OBJECT_AHG, 0x060030F0, D_80A71F08 },
+    { OBJECT_AHG, 0x06005508, NULL },       { OBJECT_AHG, 0x06005728, NULL },
+    { OBJECT_BBA, 0x06002948, D_80A71F18 }, { OBJECT_CNE, 0x06001300, NULL },
+    { OBJECT_CNE, 0x06002860, NULL },       { OBJECT_BJI, 0x06002560, D_80A71F20 },
+    { OBJECT_BJI, 0x06003F68, NULL },       { OBJECT_COB, 0x06001300, NULL },
+};
+
+typedef struct {
+    /* 0x00 */ s16 objectId;
+    /* 0x04 */ FlexSkeletonHeader* unk_4;
+} EnHyUnknownStruct2; // size = 0x8
+
+static EnHyUnknownStruct2 D_80A72010[] = { { OBJECT_AOB, 0x060000F0 }, { OBJECT_BOB, 0x060000F0 },
+                                           { OBJECT_BOJ, 0x060000F0 }, { OBJECT_AHG, 0x060000F0 },
+                                           { OBJECT_BBA, 0x060000F0 }, { OBJECT_CNE, 0x060000F0 },
+                                           { OBJECT_BJI, 0x060000F0 }, { OBJECT_COB, 0x060021F8 } };
```

Comment:
The formatter will maintain a better vertical formatting if you leave a trailing comma in multi-line arrays like this one.
```suggestion
                                           { OBJECT_BJI, 0x060000F0 }, { OBJECT_COB, 0x060021F8 }, };
```